bukkit/src/main/java/de/maxhenkel/voicechat/config/ServerConfig.java

```diff
@@ -30,79 +30,79 @@ public ServerConfig(ConfigBuilder builder) {
                 );
         voiceChatCodec = builder
                 .enumEntry("codec", Codec.VOIP,
-                        "The opus codec"
+                        "The Opus codec",
+                        "Valid values are 'VOIP', 'AUDIO', and 'RESTRICTED_LOWDELAY'"
                 );
         voiceChatMtuSize = builder
                 .integerEntry("mtu_size", 1024, 256, 10000,
-                        "The maximum size in bytes in a voice packet",
-                        "Set this to a lower value if your voice packets don't arrive"
+                        "The maximum size that voice packets are allowed to have (in bytes)"
                 );
         keepAlive = builder
                 .integerEntry("keep_alive", 1000, 1000, Integer.MAX_VALUE,
-                        "The frequency in which keep alive packets are sent",
+                        "The frequency at which keep-alive packets are sent (in milliseconds)",
                         "Setting this to a higher value may result in timeouts"
                 );
         groupsEnabled = builder
                 .booleanEntry("enable_groups", true,
-                        "If group chats are allowed"
+                        "Controls whether group chats are allowed"
                 );
         voiceHost = builder
                 .stringEntry("voice_host", "",
-                        "The host name that clients should use to connect to the voice chat",
+                        "The hostname that clients should use to connect to the voice chat",
                         "This may also include a port, e.g. 'example.com:24454'",
-                        "Don't change this value if you don't know what you are doing"
+                        "Do NOT change this value if you don't know what you're doing"
                 );
         allowRecording = builder
                 .booleanEntry("allow_recording", true,
-                        "If players are allowed to record the voice chat"
+                        "Controls whether players are allowed to record audio"
                 );
         spectatorInteraction = builder
                 .booleanEntry("spectator_interaction", false,
-                        "If spectators are allowed to talk to other players"
+                        "Controls whether spectators are allowed to talk to other players"
                 );
         spectatorPlayerPossession = builder
                 .booleanEntry("spectator_player_possession", false,
-                        "If spectators can talk to players they are spectating"
+                        "Controls whether spectators can talk to players they are spectating"
                 );
         forceVoiceChat = builder
                 .booleanEntry("force_voice_chat", false,
-                        "If players without the mod should get kicked from the server"
+                        "Controls whether players without this mod should be kicked from the server"
```

Comment:
This mod doesn't make sense on the server, since they could also use the plugin

Reply:
That's a good point, thanks! I'll make some adjustments here.